ur_ros2_control_demos/launch/ur5e_run_move_group.launch.py

```diff
@@ -28,26 +30,76 @@ def load_yaml(package_name, file_path):
 
 def generate_launch_description():
 
-    # planning_context
-    robot_description_path = os.path.join(
+    # set ur robot
+    robot_name = 'ur5e'
+
+    # <robot_name> parameters files
+    joint_limits_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                       robot_name, 'joint_limits.yaml')
+    kinematics_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                     robot_name, 'default_kinematics.yaml')
+    physical_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                   robot_name, 'physical_parameters.yaml')
+    visual_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                 robot_name, 'visual_parameters.yaml')
+
+    # common parameters
+    # If True, enable the safety limits controller
+    safety_limits = False
+    # The lower/upper limits in the safety controller
+    safety_pos_margin = 0.15
+    # Used to set k position in the safety controller
+    safety_k_position = 20
+
+    use_ros2_control = True
+
+    script_filename = os.path.join(
+        get_package_share_directory('ur_robot_driver'),
+        'resources',
+        'ros_control.urscript')
```

Comment:
For the future: This sequence is commonly repeated. It would make sense to put it in an external `.launch.py` file.